bin/ubuntu-core-initramfs

```diff
@@ -73,6 +73,22 @@ def create_initrd(parser, args):
                             ]
                         )
 
+        subprocess.check_call(
+            [
+                "/usr/lib/dracut/dracut-install",
+                "-D",
+                main,
+                "-r", kernel_root,
+                "--kerneldir",
+                modules,
+                "--firmwaredirs",
+                firmware,
+                "--module",
+                "--optional",
+                "=drivers/hid",
```

Comment:
How much bigger is the initramfs after adding this modules?

Reply:
Approximately 1.4M more for x86_64 (from ~27M to ~28M).